src/summary_writer.jl

```diff
@@ -28,6 +28,7 @@ Arguments:
 * graph: A `Graph` object.
 """
 function FileWriter(log_dir::AbstractString; graph=get_def_graph())
+    mkdir(log_dir)
```

Comment:
This needs to check the that the directory does not already exist before creating it, or it will throw  an error.
Possibly the behavor `mkpath` is more intuitive for a user, 
that doesn't throw an error if any path of the path exists.